### DISCUSSION TITLE
Copilot still free for github verified students?

### DISCUSSION THREAD

**Discussion by avipars** [community/community#19471] [Copilot Conversations] [answered]
Hello, 

is Copilot still going to be free for verified students or will they have to pay the going monthly rate going forward?

Thank you,
Avi

**Comment by melvincwng** [comment, marked as answer]
@avipars, GitHub Copilot is free to use for verified students!

Hope this clarifies

**Reply by melvincwng** [reply]
@avipars, please kindly 'Mark as answer' if I have helped addressed your question.

Thanks for your cooperation and have a great day ahead 😄

**Reply by avipars** [reply]
Thank you so much! I was worried because I got the same email they sent to everyone about changing the model to subscription...

Will make as closed!

**Reply by melvincwng** [reply]
@avipars, pleasure to have helped you!
